dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/Wrapper.java

```diff
@@ -1,453 +1,453 @@
-     * get property type.
-     *
-     * @param pn property name.
-     * @return Property type or nul.
-     */
-    abstract public Class<?> getPropertyType(String pn);
-
-    /**
-     * has property.
-     *
-     * @param name property name.
-     * @return has or has not.
-     */
-    abstract public boolean hasProperty(String name);
-
-    /**
-     * get property value.
-     *
-     * @param instance instance.
-     * @param pn       property name.
-     * @return value.
-     */
-    abstract public Object getPropertyValue(Object instance, String pn) throws NoSuchPropertyException, IllegalArgumentException;
-
-    /**
-     * set property value.
-     *
-     * @param instance instance.
-     * @param pn       property name.
-     * @param pv       property value.
-     */
-    abstract public void setPropertyValue(Object instance, String pn, Object pv) throws NoSuchPropertyException, IllegalArgumentException;
-
-    /**
-     * get property value.
-     *
-     * @param instance instance.
-     * @param pns      property name array.
-     * @return value array.
-     */
-    public Object[] getPropertyValues(Object instance, String[] pns) throws NoSuchPropertyException, IllegalArgumentException {
-        Object[] ret = new Object[pns.length];
-        for (int i = 0; i < ret.length; i++) {
-            ret[i] = getPropertyValue(instance, pns[i]);
-        }
-        return ret;
-    }
-
-    /**
-     * set property value.
-     *
-     * @param instance instance.
-     * @param pns      property name array.
-     * @param pvs      property value array.
-     */
-    public void setPropertyValues(Object instance, String[] pns, Object[] pvs) throws NoSuchPropertyException, IllegalArgumentException {
-        if (pns.length != pvs.length) {
-            throw new IllegalArgumentException("pns.length != pvs.length");
-        }
-
-        for (int i = 0; i < pns.length; i++) {
-            setPropertyValue(instance, pns[i], pvs[i]);
-        }
-    }
-
-    /**
-     * get method name array.
-     *
-     * @return method name array.
-     */
-    abstract public String[] getMethodNames();
-
-    /**
-     * get method name array.
-     *
-     * @return method name array.
-     */
-    abstract public String[] getDeclaredMethodNames();
-
-    /**
-     * has method.
-     *
-     * @param name method name.
-     * @return has or has not.
-     */
-    public boolean hasMethod(String name) {
-        for (String mn : getMethodNames()) {
-            if (mn.equals(name)) {
-                return true;
-            }
-        }
-        return false;
-    }
-
-    /**
-     * invoke method.
-     *
-     * @param instance instance.
-     * @param mn       method name.
-     * @param types
-     * @param args     argument array.
-     * @return return value.
-     */
-    abstract public Object invokeMethod(Object instance, String mn, Class<?>[] types, Object[] args) throws NoSuchMethodException, InvocationTargetException;
-}
+/*
```

Comment:
Please recover the file end-line format of this file.

Reply:
Strange, github will automatically add a empty line